config/locales/en.yml

```diff
@@ -796,6 +797,8 @@ en:
     order_receipt:
       subject: "NU Core Order Receipt"
       intro: "Thank you for your order on the NU Core website."
+      label:
+        ordered_at: Ordered at
```

Comment:
This is here to preserve the wording on the mailed receipts before I localized them. Everywhere else it looks like `OrderDetail#ordered_at` humanizes to "Ordered Date".